src/utils.py

```diff
@@ -31,6 +32,14 @@ def __init__(self):
     def get_disk(self):
         return (str(int(psutil.disk_usage('/').total/ (1024 ** 3))))
 
+    def get_disks(self):
+        context = pyudev.Context()
+        disks = {}
+        for device in context.list_devices(subsystem='block', DEVTYPE='disk'):
+            if not 'loop' in device['DEVNAME']:
```

Comment:
With the switch to pyudev for this, it's also reporting CD/DVD drives as disks which causes a false position. Please also filter out device names that match sr0 too. The first CD/DVD drive will show as sr0, and we really don't need to worry about kramden devices having multiple CD/DVD drives.